geniesp/bpc_redcap_export_mapping.py

```diff
@@ -561,11 +564,26 @@ def genie_clinicaldf(self) -> pd.DataFrame:
         Returns:
             pd.DataFrame: main GENIE information for sponsored project samples
         """
-        genie_clinicaldb = self.syn.tableQuery(
-            f"select SAMPLE_ID, PATIENT_ID, ONCOTREE_CODE, SEQ_ASSAY_ID, "
-            f"SAMPLE_TYPE, SEQ_YEAR from {self._CLINICAL_SYNID}"
-        )
-        genie_clinicaldf = genie_clinicaldb.asDataFrame()
+        # genie_clinicaldb = self.syn.tableQuery(
+        #     f"select SAMPLE_ID, PATIENT_ID, ONCOTREE_CODE, SEQ_ASSAY_ID, "
+        #     f"SAMPLE_TYPE, SEQ_YEAR from {self._CLINICAL_SYNID}"
+        # )
+        # genie_clinicaldf = genie_clinicaldb.asDataFrame()
+        # DECISION 06/2023: use the release file to do retractions
+        # This is due to the most recent releases potentially having
+        # samples retracted. The consortium release matched with the
+        # public release (14.7-consortium <-> 14.0-public) must be used
+        # due to SEQ_YEAR being used through the code.
+        sample_synid = self.get_mg_synid(
+            self._MG_RELEASE_SYNID, "data_clinical_sample.txt"
+        )
+        genie_clinicaldf = pd.read_csv(
+            self.syn.get(sample_synid, followLink=True).path, sep="\t", comment="#"
+        )
+        # Filter out cfDNA samples
```

Comment:
Determine impact across all cohorts for removing cfDNA samples

Reply:
https://sagebionetworks.jira.com/jira/software/c/projects/GEN/issues/GEN-876